ci/run.sh

```diff
@@ -108,7 +116,7 @@ run_one () {
     # Replace all forward slashes with '-'
     CONTAINER_FOR_TAG="${CONTAINER_FOR_TAG//\//-}"
 
-    TAG="shadow:$CONTAINER_FOR_TAG-$CC-$BUILDTYPE"
+    TAG="$REPO:$CONTAINER_FOR_TAG-$CC-$BUILDTYPE"
```

Comment:
There's a reference to the "shadow" tag name in https://shadow.github.io/docs/guide/ci.html which should maybe be updated to say "shadowsim/shadow-ci".

Reply:
Done